examples/3d/anti_aliasing.rs

```diff
@@ -28,6 +28,13 @@ fn main() {
         .run();
 }
 
+type TaaComponents = (
+    TemporalAntiAliasing,
+    TemporalJitter,
+    DepthPrepass,
+    MotionVectorPrepass,
+);
```

Comment:
This is kinda scuffed, but component removal is a bit more annoying without bundles 🤔

(of course I could also make this an actual `Bundle` like `TemporalAntiAliasBundle` too, but the properties aren't needed in this case)

Reply:
It does seem like we're missing a `camera.remove_with_requires::<TemporalAntiAliasing>()` function in bevy_ecs

Reply:
Tracked in #15580.